### PR TITLE
Add anew route to show the version number.

### PR DESCRIPTION
**Pull request by neonleann**
https://trello.com/c/sTTPuoVb/1771-add-a-version-page-that-returns-the-version-number-to-service

### CODE REVIEW

File: app/Http/Controllers/Service/VersionController.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    public function index()
+    {
+        // accept_cookies are deprecated, remove for anyone who has one set
+        return response(view('welcome'))->cookie(Cookie::forget('accept_cookies'));
+    }
+
+    public function version(): JsonResponse
+    {
+        /** @noinspection PhpComposerExtensionStubsInspection */
+        $pkg = json_decode(file_get_contents(base_path('package.json')), false);
```
Comment (charlesstrange2) [inline app/Http/Controllers/Service/VersionController.php:20]: can we pull from composer.json (to avoid multiple version number places)

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Service/VersionController.php (hunk 1 above)
Comment (neonleann) [reply app/Http/Controllers/Service/VersionController.php]: Done.

File: package.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  "version": "1.13",
```
[the remark this answers is not in the document]
Comment (neonleann) [reply package.json]: Done.

File: app/Http/Controllers/Service/VersionController.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace App\Http\Controllers\Service;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Cookie;
+
+class VersionController extends Controller
+{
+    public function index()
```
[the remark this answers is not in the document]
Comment (neonleann) [reply app/Http/Controllers/Service/VersionController.php]: Removed.

File: routes/service.php [hunk 4]
```diff
@@ -15,6 +15,11 @@
 
 Route::get('/', 'AdminController@index')->name('admin.dashboard');
 
+Route::get('version', [
```
[the remark this answers is not in the document]
Comment (neonleann) [reply routes/service.php:18]: Moving it to /web.php (where there's currently nothing, btw) breaks everything. So I left it as it is.

[the remark this answers is not in the document]
Comment (charlesstrange2) [reply routes/service.php:18]: most wise.

**Review by charlesstrange2 — APPROVED** [review]
👍

**Pull request merged**